lib/dal/src/schema/variant/leaves.rs

```diff
@@ -80,6 +82,7 @@ impl Into<RootPropChild> for LeafInputLocation {
             LeafInputLocation::Domain => RootPropChild::Domain,
             LeafInputLocation::Resource => RootPropChild::Resource,
             LeafInputLocation::DeletedAt => RootPropChild::DeletedAt,
+            LeafInputLocation::Secrets => RootPropChild::Secrets,
```

Comment:
Ah, this is how, nice! I was so wrong that the solution would likely need SQL query changes.